.semaphore/semaphore.yml

```diff
@@ -110,3 +84,7 @@ promotions:
     pipeline_file: multi-arch-builds-and-upload.yml
     auto_promote:
       when: "result = 'passed' and branch =~ '.*'"
+  - name: "Third Party Notices PR"
+    pipeline_file: third-party-notices-pr.yml
```

Comment:
Hm, let's either change this to `third-party-notices.yml` or rename the file to `third-party-notices-pr.yml`. I would vote for latter but anything works.